scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -302,6 +291,20 @@ <h5 class="modal-title">Remove gene panel "{{panel.display_name}}"?</h5>
     }
 });
 
+function toggleRow(hideCheck) {
+  var els = document.getElementsByClassName("hidableRow");
+  for(var i=0;i<els.length;i++){
+    collapse = bootstrap.Collapse.getOrCreateInstance(els[i])
+    if(hideCheck.checked){
+      collapse.hide();
```

Comment:
This is the change here: using bootstrap collapse instead of the html style that is not fully implemented on all browsers.